charts/statping/templates/common.yaml

```diff
@@ -0,0 +1,26 @@
+{{/* Make sure all variables are set properly */}}
+{{- include "common.values.setup" . }}
+
+{{/* use the port specified in the service */}}
+{{- $_ := set .Values.env "VIRTUAL_PORT" .Values.service.port.port -}}
+
+{{/* if DB_HOST is not set use internal postgres chart */}}
+{{- if eq .Values.env.DB_HOST "" -}}
+  {{- $_ := set .Values.env "DB_HOST" (printf "%s-postgresql" .Release.Name) -}}
+{{- end -}}
+{{/* if TTRSS_DB_USER is not set use internal postgres chart */}}
+{{- if eq .Values.env.DB_USER "" -}}
+  {{- $_ := set .Values.env "DB_USER" .Values.postgresql.postgresqlUsername -}}
+{{- end -}}
+{{/* if TTRSS_DB_USER is not set use internal postgres chart */}}
+{{- if eq .Values.env.DB_PASS "" -}}
+  {{- $_ := set .Values.env "DB_PASS" .Values.postgresql.postgresqlPassword -}}
+{{- end -}}
+{{/* if TTRSS_DB_USER is not set use internal postgres chart */}}
+{{- if eq .Values.env.DB_DATABASE "" -}}
+  {{- $_ := set .Values.env "DB_DATABASE" .Values.postgresql.postgresqlDatabase -}}
+{{- end -}}
```

Comment:
I'm pretty sure there was some copy/pasting going on here from the TTRSS chart 😉

Reply:
Indeed :-D